src/main/resources/assets/gregtech/lang/en_us.lang

```diff
@@ -861,6 +861,10 @@ cover.conveyor.title=Conveyor Cover Settings (%s)
 cover.conveyor.transfer_rate=%s items/sec
 cover.conveyor.mode.export=Mode: Export
 cover.conveyor.mode.import=Mode: Import
+cover.conveyor.distribution.enabled=Item distribution mode for item pipes/n§bInsert First
+cover.conveyor.distribution.disabled=Item distribution mode for item pipes/n§bRound Robin
+cover.conveyor.blocks_input.enabled=If enabled, items will not be inserted when cover is set to put items from cover holder to pipe./n§aEnabled
```

Comment:
I reccomend rephrasing this slightly to: "to pull items from the cover holder into pipes.

Reply:
will rename "cover holder" to "inventory" because the cover holder can be the pipe itself